src/main/resources/public/locales/ozwillo/fr/messages.json

```diff
@@ -1,8 +1,8 @@
 {
-  "<0>{email}</0> will be invited to the organization": "<0>{email}</0> sera invité à l'organisation",
+  "<0>{email}</0> will be invited to the organization": "<0>{email}</0> sera invité à la collectivité",
```

Comment:
the modifications in this file are specific to the SICTIAM and will have to be put in a specific translation file for the SICTIAM.